### PR TITLE
DL-4174 Update logging config

### PR DESCRIPTION
**Pull request by CameronStamp**
DL-4174 Update logging config

Update

Update logging in xml files

## Checklist

*Reviewee* (Replace with your name)
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

*Reviewer* (Replace with your name)
 - [x]  I've confirmed that every effort has been made to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've confirmed appropriate tests has been included with any code added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've confirmed code was added using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: conf/logback.xml [hunk 1]
```diff
@@ -43,27 +43,24 @@
...
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.channel.DefaultChannelPool" level="WARN"/>
-    <logger name="play.core.netty.utils.ServerCookieDecoder" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.request.NettyRequestSender" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.handler.HttpHandler" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.channel.ChannelManager" level="WARN"/>
-    <logger name="org.asynchttpclient.netty" level="WARN"/>
+    <logger name="com.google.inject" level="WARN"/>
 
-    <logger name="uk.gov" level="INFO"/>
+    <logger name="uk.gov" level="WARN"/>
```
Comment (paulanderson1234) [inline conf/logback.xml:48]: According to acceptance criteria should this not remain as INFO ?
'Set the logger levels in logback.xml to the same as what we had set for app-config-production'

File: conf/logback.xml [hunk 2]
```diff
@@ -43,27 +43,24 @@
...
 
-    <logger name="application" level="DEBUG"/>
+    <logger name="application" level="WARN"/>
```
Comment (paulanderson1234) [inline conf/logback.xml:50]: According to acceptance criteria should this not remain as INFO ?
'Set the logger levels in logback.xml to the same as what we had set for app-config-production'

File: conf/logback.xml [hunk 3]
```diff
@@ -43,27 +43,24 @@
...
 
-    <logger name="com.google.inject" level="INFO"/>
-
-    <logger name="connector" level="TRACE">
+    <logger name="connector" level="WARN">
```
Comment (paulanderson1234) [inline conf/logback.xml:52]: According to acceptance criteria should this not remain as INFO ?
'Set the logger levels in logback.xml to the same as what we had set for app-config-production'

File: conf/logback.xml [hunk 4]
```diff
@@ -43,27 +43,24 @@
...
         <appender-ref ref="STDOUT"/>
     </logger>
 
-    <root level="DEBUG">
+    <logger name="com.codahale.metrics.graphite.GraphiteReporter" level="OFF" />
```
Comment (paulanderson1234) [inline conf/logback.xml:56]: Re all of these set as WARN:
According to acceptance criteria should this not remain as INFO ?
'Set the logger levels in logback.xml to the same as what we had set for app-config-production'

File: conf/logback.xml [hunk 5]
```diff
@@ -43,27 +43,24 @@
...
 
-    <logger name="com.ning.http.client" level="WARN"/>
-    <logger name="org.asynchttpclient.netty.channel.DefaultChannelPool" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.channel.DefaultChannelPool" level="WARN"/>
-    <logger name="play.core.netty.utils.ServerCookieDecoder" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.request.NettyRequestSender" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.handler.HttpHandler" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.channel.ChannelManager" level="WARN"/>
-    <logger name="org.asynchttpclient.netty" level="WARN"/>
+    <logger name="com.google.inject" level="WARN"/>
```
Comment (paulanderson1234) [inline conf/logback.xml:46]: Why are we adding this specifically - not mentioned in acceptance criteria and haven't seen it in other repos we have changed recently?

Comment (CameronStamp) [reply conf/logback.xml:46]: So I added this as was in the example for step 2 for the logback file https://github.com/hmrc/vat-summary-frontend/blob/master/conf/logback.xml though I may have misinterpreted which needed adding removing there.

File: conf/logback.xml (hunk 1 above)
Comment (CameronStamp) [reply conf/logback.xml:48]: Yes, should be INFO, i've copied a section from https://github.com/hmrc/vat-summary-frontend/blob/master/conf/logback.xml and then forgotten to change back to INFO

File: conf/logback.xml (hunk 3 above)
Comment (CameronStamp) [reply conf/logback.xml:52]: Yes, should be INFO, i've copied a section from https://github.com/hmrc/vat-summary-frontend/blob/master/conf/logback.xml and then forgotten to change back to INFO

File: conf/logback.xml (hunk 4 above)
Comment (CameronStamp) [reply conf/logback.xml:56]: Yes, should be INFO, i've copied a section from https://github.com/hmrc/vat-summary-frontend/blob/master/conf/logback.xml and then forgotten to change back to INFO

File: conf/logback.xml (hunk 3 above)
Comment (CameronStamp) [reply conf/logback.xml:52]: Yes, should be INFO, i've copied a section from https://github.com/hmrc/vat-summary-frontend/blob/master/conf/logback.xml and then forgotten to change back to INFO

File: conf/logback.xml (hunk 2 above)
Comment (CameronStamp) [reply conf/logback.xml:50]: Yes, should be INFO, i've copied a section from https://github.com/hmrc/vat-summary-frontend/blob/master/conf/logback.xml and then forgotten to change back to INFO

File: conf/logback.xml (hunk 5 above)
Comment (paulanderson1234) [reply conf/logback.xml]: Happy to leave in as it is present in other repos and is not affecting functionality

**Pull request merged**